daprdocs/content/en/dotnet-sdk-docs/dotnet-workflow/dotnet-workflowclient-usage.md

```diff
@@ -0,0 +1,70 @@
+---
+type: docs
+title: "DaprWorkflowClient usage"
+linkTitle: "DaprWorkflowClient usage"
+weight: 100000
+description: Essential tips and advice for using DaprWorkflowClient
+---
+
+## Lifetime management
+
+A `DaprWorkflowClient` holds access to networking resources in the form of TCP sockets used to communicate with the Dapr sidecar as well
+as other types used in the management and operation of Workflows. `DaprWorkflowClient` implements `IAsyncDisposable` to support eager
+cleanup of resources.
+
+## Dependency Injection
+
+The `AddDaprWorkflow()` method will register the Dapr workflow services with ASP.NET Core dependency injection. This method
+requires an options delegate that defines each of the workflows and activities you wish to register and use in your application.
+
+This method will also register a `DaprClient` instance as it's used to communicate with the Dapr sidecar and the lifetime options
+provided below will be used for that registration and its own dependencies as well.
```

Comment:
Will *attempt* to use, if it hasn't already been registered with some other lifetime.  (E.g. an earlier call to `AddDaprClient()` with a singleton lifetime will use a single instance of the `DaprClient` regardless of the lifetime chosen for the workfloaw client).

Reply:
Excellent point to call out. Rewrote that little section and added a note calling it out.